index.js

```diff
@@ -722,24 +171,9 @@ try {
   return;
 }
 
-  // write cyper intructions to file
-fs.writeFileSync('./data/import.cypher', instructions.join('\n'), 'utf8');
-
-  // ========================================================================
-  // write a smaller version of the hpa rna levels file, to send to the frontend
-  // remove expressions of genes not in any human models parsed
-if (!fs.existsSync(`${inputDir}/hpaRnaFull.json`)) {
-    throw new Error("HPA rna JSON file not found");
-} else {
-  const hpaRnaExpressionJson = require(`${inputDir}/hpaRnaFull.json`);
-
-  Object.keys(hpaRnaExpressionJson.levels).forEach((geneId) => {
-    if (!humanGeneIdSet.has(geneId)) {
-      delete hpaRnaExpressionJson.levels[geneId];
-    }
-  });
-
-  const json_rna = JSON.stringify(hpaRnaExpressionJson);
-  fs.writeFileSync('./data/hpaRna.json', json_rna);
-}
+// ========================================================================
+// write cypher intructions to file
+writer.writeCypherFile(instructions, outDir);
 
+// write a smaller version of the hpa rna levels file, to send to the frontend
+writer.writeHpaRnaJson(humanGeneIdSet, inputDir, outDir);
```

Comment:
It looks like `humanGeneIdSet` is not being set, so I think the output would be empty.